simplified-app-simplye/src/main/assets/Accounts.json

```diff
@@ -1357,17 +1397,17 @@
         "supportsSimplyESync": true
     },
     {
-        "authPasscodeAllowsLetters": true,
         "cardCreatorUrl": "https://archive.org/account/login.createaccount.php",
         "catalogUrl": "https://bookserver.archive.org/simple",
         "id_numeric": 5,
         "id_uuid": "urn:uuid:f60b644e-4955-4996-a4e5-a192feb4e7f8",
         "inProduction": false,
+        "loginKeyboard": "Email address",
```

Comment:
This is the motivating case behind this change -- the Internet Archive is telling us to use the email keyboard when asking people to log in.